pylint/checkers/utils.py

```diff
@@ -1264,6 +1265,9 @@ def get_node_last_lineno(node: astroid.node_classes.NodeNG) -> int:
 
 def is_postponed_evaluation_enabled(node: astroid.node_classes.NodeNG) -> bool:
     """Check if the postponed evaluation of annotations is enabled"""
+    if sys.version_info[:2] >= (3, 10):
```

Comment:
Can you move this in a constant at the module level? e.g. `PY310 = sys.version_info[:2] >= (3, 10)` then we use the value in the function body.

Reply:
Applied.